### PR TITLE
Image | getPreviewContainer

### PR DESCRIPTION
**Pull request by rfreling**

### CODE REVIEW

File: src/Image.tsx [hunk 1]
```diff
@@ -16,6 +16,7 @@ export interface ImageProps
   preview?: boolean;
   onPreviewClose?: (e: React.SyntheticEvent<HTMLDivElement | HTMLLIElement>) => void;
   onClick?: (e: React.MouseEvent<HTMLDivElement>) => void;
+  getPreviewContainer?: (node: HTMLElement) => HTMLElement;
```
Comment (shaodahong) [inline src/Image.tsx:19]:
```suggestion
  getPreviewContainer?: () => HTMLElement;
```

Comment (rfreling) [reply src/Image.tsx]: 🙏  @shaodahong

**Comment by zombieJ** [comment]
Pls use `getPopupContainer` as naming instead.

**Comment by rfreling** [comment]
🙏 @zombieJ  Just updated – 谢谢!

**Comment by zombieJ** [comment]
`+ rc-image@3.2.0`

**Pull request merged**